### PR TITLE
added loaders to graphs

### PR DESCRIPTION
**Pull request by karthikgeek1**

### CODE REVIEW

File: src/Components/Graphs/Graph1.jsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 import "./Graph1.css";
 
 function Graph1() {
   const [data, setdata] = useState([]);
+  const [loader, setLoader] = useState(false)
+
+  useEffect(() => {
+    setLoader(true)
+    setTimeout(() => {
+      setLoader(false)
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph1.jsx:15]: What if data takes more than 5 secs to load due to slow connection?

File: src/Components/Graphs/Graph2.jsx [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 
 const DemoLine = () => {
   const [data, setdata] = useState([]);
+  const [loader, setLoader] = useState(false)
+
+  useEffect(() => {
+    setLoader(true)
+    setTimeout(() => {
+      setLoader(false)
+    }, 5000)
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph2.jsx:16]: same as above

File: src/Components/Graphs/Graph3.jsx [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
 import "./Graph3.css";
 
 const DemoArea = () => {
   const [data, setdata] = useState([]);
+  const [loader, setLoader] = useState(false)
+
+  useEffect(() => {
+    setLoader(true)
+    setTimeout(() => {
+      setLoader(false)
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph3.jsx:15]: same as above

File: src/Components/Layout/StockLayout.jsx [hunk 4]
```diff
@@ -1,7 +1,5 @@
 import { Layout } from "antd"
 
-import { TableComponent } from "../table/TableComponent";
-
 import "../Layout/StockLayout.css";
 
 import React, {useState, useEffect, useContext} from "react";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx:3]: format imports

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Graphs/Graph1.jsx (hunk 1 above)
Comment (karthikgeek1) [reply src/Components/Graphs/Graph1.jsx:15]: how to solve this sir?

Comment (Souvikdeb2612) [reply src/Components/Graphs/Graph1.jsx:15]: set loader to false after data fetching is done

Comment (karthikgeek1) [reply src/Components/Graphs/Graph1.jsx:16]: ok sir

Comment (karthikgeek1) [reply src/Components/Graphs/Graph1.jsx:15]: I tried it sir, but it was not working.

**Pull request closed without merging**